### ISSUE TITLE
Marp won't open the editor/preview window - OSX 10.9

### ISSUE THREAD

**Issue by neilstuartcraig** [yhatt/marp#58]
Hi (again)

I had a working OSX 10.9 install of Marp and created a couple of docs using it. The next day though (no reboot but i have tried a reboot since), i found Marp no longer opens the editor/preview window. The "About" dialogue opens and the main menu works.

Is there any debug/log info I can provide? I can't see anything but happy to help/investigate.

Cheers

**Comment by yhatt** [comment]
Did you mean, does the similar issue #47 helps you?

**Comment by neilstuartcraig** [comment]
Yes! Thanks so much @yhatt - I edited my `config.json` and now Marp works perfectly again.

I saw that my config contained:

```
"windowPosition":{"width":0,"height":0,"maximized":false,"x":-1616,"y":474}
```

So i just changed it to:

```
"windowPosition":{"width":100,"height":100,"maximized":false,"x":-1616,"y":474}
```

And I could then drag-resize the window and it was all fine.

I'll add a note to #47 too.

Cheers

**Comment by yhatt** [comment]
OK. I'll close this issue. We'll fix this issue on #47.
